projects/vir/src/app/+home/home.component.html

```diff
@@ -34,6 +34,8 @@ <h1>{{ 'home.authority.title' | translate }}</h1>
       <lu-button anchor="https://cdn.laji.fi/files/vir/vir_kirjautumisohje.pdf" target="_blank"><i class="glyphicon glyphicon-file"></i> {{ "vir.home.loginInstructions.btn" | translate }}</lu-button>
       <p class="mt-5" translate>vir.home.agreement</p>
       <lu-button anchor="https://cdn.laji.fi/files/vir/vir_mallisopimus.pdf" target="_blank"><i class="glyphicon glyphicon-file"></i> {{ "vir.home.agreement.btn" | translate }}</lu-button>
+      <p class="mt-5" translate>vir.home.changelog</p>
+      <lu-button [anchor]="['/about/6669']" target="_self"><i class="glyphicon glyphicon-file"></i> {{ "vir.home.changelog.btn" | translate }}</lu-button>
```

Comment:
The link ("anchor") is missing localization. Use the `localize` pipe like you did for `projects/laji/src/app/+haseka/haseka.component.html`

Reply:
Thanks, should be fixed now.